src/main/java/com/faforever/gw/services/messaging/lobby_server/LobbyService.java

```diff
@@ -172,8 +173,8 @@ private void enqueue(OutgoingLobbyMessage message) {
         }
     }
 
-    public void createGame(UUID battleId) {
-        enqueue(new CreateGameMessage(UUID.randomUUID(), battleId));
+    public void createGame(UUID battleId, List<Long> participants) {
+        enqueue(new CreateGameMessage(UUID.randomUUID(), battleId, participants));
```

Comment:
Yes, this isn't final but getting data flowing makes things easier to work with.